lib/wickedbase.pm

```diff
@@ -260,6 +382,14 @@ sub do_mutex {
     }
 }
 
+=head2 done
+
+  done()
```

Comment:
I think it is too much , documentation of this method in place where it's declared is good enough